lib/Maintenance/Tasks/TaskHelper.php

```diff
@@ -0,0 +1,75 @@
+<?php
+declare(strict_types=1);
+
+namespace Pimcore\Maintenance\Tasks;
+
+use Pimcore\Db;
+use Pimcore\Model\DataObject\ClassDefinition;
+use Psr\Log\LoggerInterface;
+
+final class TaskHelper
+{
+    private static ?TaskHelper $instance = null;
+
+    private LoggerInterface $logger;
+    public static function create(LoggerInterface $logger): self
+    {
+        if (!self::$instance) {
+            self::$instance = new self();
+        }
+        self::$instance->logger = $logger;
+
+        return self::$instance;
+    }
+    
+    public function getDataStructureNamesMapLowerToCamelCase(string $dir): array
+    {
+        $mapLowerToCamelCase = [];
+        $files = array_diff(scandir($dir), ['..', '.']);
+        foreach ($files as $file) {
+            $classname = str_replace('.php', '', $file);
+            $mapLowerToCamelCase[strtolower($classname)] = $classname ;
+        }
+
+        return $mapLowerToCamelCase;
+    }
+
+    public function getClassDefintionByClassId(string $classId, string $tableName): ClassDefinition|null
+    {
+        $classDefinition = ClassDefinition::getByIdIgnoreCase($classId);
+        if (!$classDefinition) {
+            $this->logger->error("Classdefinition '" . $classId . "' not found. Please check table " . $tableName);
+        }
+        return $classDefinition;
+    }
+
+    public function cleaningTable(
```

Comment:
```suggestion
    public function cleanupTable(
```
seems more accurate like this..